packages/cli/src/commands/webpack.js

```diff
@@ -0,0 +1,92 @@
+import util from 'util'
+import consola from 'consola'
+import get from 'lodash/get'
+import { common } from '../options'
+
+export default {
+  name: 'webpack',
+  description: 'Inspect webpack config',
+  usage: 'webpack [path] [prop=value]',
+  options: {
+    ...common,
+    name: {
+      alias: 'n',
+      type: 'string',
+      default: 'Client',
```

Comment:
Could we make this case insensitive? I rly dont want to have to remember to type Client instead of client